### DISCUSSION TITLE
connection

### DISCUSSION THREAD

**Discussion by alonelove7** [vpnhood/VpnHood#155] [Q&A] [answered]
Hello sir
how can i set how many user can connect to server?
and in default run https://github.com/vpnhood/VpnHood/wiki/How-to-run-VpnHood-Server-on-Linux
how many users can connect?

thank you

**Comment by trudyhood** [comment]
If you are running your own server without our AccessServer use the following command and set MaxClient
`dotnet VpnHoodServer.dll gen -?`

You need to control the number of tokens you distribute.

**Comment by alonelove7** [comment, marked as answer]
so -?
that command will create token , right ?
or the number of connection
---

**Reply by trudyhood** [reply]
It creates a token, and you can specify the number of connections for that token

**Reply by alonelove7** [reply]
Thank you.
